.changeset/orange-vans-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Replace acorn/acorn-walk with esbuild
```

Comment:
Could you follow the [changeset message format](https://github.com/cloudflare/wrangler2/blob/main/CONTRIBUTING.md#changeset-message-format) here and also mention pages too, please?

Reply:
Yes, sorry about that.